pyEpiabm/pyEpiabm/core/microcell.py

```diff
@@ -118,6 +120,23 @@ def add_place(self, n: int, loc: typing.Tuple[float, float],
             self.cell.places.append(p)
             self.places.append(p)
 
+    def add_household(self, people: list):
+        """Adds a default :class:`Household` to Microcell and fills it with
+        a number of :class:`Person` s.
+
+        Parameters
+        ----------
+        people : list
+            List of default :class:`People` to add to household
+
+        """
+        household = Household(self, loc=(1.0, 1.0))
+        household.loc = self.location
```

Comment:
Just wondering if we send the loc in the init, why we also have to set the attribute.
Also do we even need `household.loc`. As opposed to `husehold.location`